Looped/AppDelegate.swift

```diff
@@ -0,0 +1,16 @@
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+    
+    lazy var window: UIWindow? = UIWindow(frame: UIScreen.main.bounds)
+    
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        
+        let tabBarController = TabBarController()
```

Comment:
Tak w ogóle to tab bar zrobilbym prywatna stala na poziomie klasy a nie funkcji.